src/cozmo/behavior.py

```diff
@@ -45,8 +47,27 @@
 from ._clad import _clad_to_engine_iface, _clad_to_engine_cozmo
 
 
+#: string: Behavior idle state (not requested to run)
+BEHAVIOR_IDLE = 'behavior_idle'
+
+#: string: Behavior requested state (waiting for engine to start it)
+BEHAVIOR_REQUESTED = 'behavior_requested'
+
+#: string: Behavior running state
+BEHAVIOR_RUNNING = 'behavior_running'
+
+#: string: Behavior succeeded state
```

Comment:
Is it possible to request that the behavior stops after it has been started? If so, then maybe this wouldn't be the succeeded state?